index.js

```diff
@@ -125,7 +126,18 @@ var publicAgent = {
     if (!config.enabled) {
       return this;
     }
-    var logger = common.logger.create(config.logLevel, '@google/cloud-trace');
+
+    var logLevel = config.logLevel || 2;
+    if (logLevel < 0) {
```

Comment:
We have a hazard here if `config.logLevel` came from the environment. Numbers would show up as strings. That's not fatal however since the expression `'4' < 0` does evaluates correctly due to type coersion `false`.

However `'monkey' < 0` is a valid comparison too that evaluates to `false`. In other words, if the user had an invalid log level, we disable logging for them.